docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,173 @@ described below, and relative to the joint TMREL.
+
+All-cause mortality is sum of all cause-specific mortalities: 
+
+   ACMR =  :math:`\sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{salmon}}\text{CSMR}`
+
+The mortality from unmodelled causes affected by lbwsg (blue) is thus: 
+
+   :math:`\sum\limits_{\text{blue}}\text{CSMR}` = ACMR - :math:`\sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR}`
+
+Because we model some of the causes affected by lbwsg, we can use their excess-morality rates (EMR) instead of the average CSMRs: The mortality from modelled causes affected by lbwsg (green):
+
+   - cause-specific mortality if the person who does NOT have the condition: 0
+   - cause-specific mortality if the person HAS the condition: EMR of the condition
+
+ 
+We are interested in applying the PAF and relative risk to only the causes that GBD considers to be affected by lbwsg (green and blue):
+
+   |  i = low birth weight short gestation category
+   |  mr_i = mortality hazard in early and late neonatal period for category i
+   |  rr_i = relative risk for all cause mortality in category i
+   |  state = either 1 with condition or 0 without condition 
+   |  PAF* = this is the PAF of the most detailed cause affected by lbwsg
+
+Hence, the mortality hazard for an individual in lbwsg category i is:  
+
+mr_i 
+
+   | = ACMR_i 
+   | = (sum of unaffected causes) + affected(sum of unmodelled + sum of modelled) x (1-PAF*) x :math:`rr_i`
+
+= :math:`\sum\limits_{\text{salmon}}\text{CSMR} + (\sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+= :math:`\sum\limits_{\text{salmon}}\text{CSMR} + (ACMR - \sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
+
+
+.. important :: 
+
+   PAF in the above equation represents PAFs for most-detailed-cause (they are all roughly equal) affected by LBWSG (or as calculated in vivarium from LBWSG relative risks and exposure). This approach assumes that relative risks for LBWSG applies only to causes that GBD considers to be affected by LBWSG (green and blue causes). 
+
+.. todo ::
+   link notebook that ali made on obtaining most-detailed-cause PAF lbwsg PAFs and abie's notebook on continuous conversion of the categories 
+
+   
+Assumptions and limitations
++++++++++++++++++++++++++++
+ 
+Strengths
+
+   o  This approach is consistent with GBD methodology and avoids artificially decreasing the mortality rate for individual causes that are not affected by improvements in LBWSG (due to reverse causality or other concerns).
+   
+Limitations
+
+   o  This approach applies the LBWSG relative risks in an inconsistent manner with what they represent (ratios of ACMRs). This implies that the relative risks may be different as they relate to the subset of causes that GBD considers affected by LBWSG rather than all causes.
```

Comment:
I had to re-read this a couple times to understand what it was saying. I think the point is that the RR's are supposed to represent _all cause_ mortality, but we are only applying them to _some_ of the causes (the ones GBD says are affected), correct?  I think it would be clearer if the "all vs. some" distinction were emphasized sooner, rather than at at the end of the comment.

Reply:
I've tried to make this clearer: In the risk appendix of GBD 2017, it says the relative risks for lbwsg apply to all-cause mortality rates. However, we are choosing to apply the RRs *only* to the causes that GBD believe lbswg affects (the list of affected causes). Our approach may be inconsistent with the GBD intended for the RRs.

Reply:
Well, I think we _are_ using the RR's consistently with what GBD intended -- we're applying them exactly to the causes GBD says to apply them to.  The potential inconsistency is from the way the data for the RR's was collected -- if our understanding is correct, these RR's were computed for _all_-cause mortality rates, but then GBD excluded all the causes for which they couldn't plausibly establish causality, i.e. everything except the blue causes.

So yes, I think your above explanation is clearer, except for the last sentence.

Reply:
ah yes-ok, I think that is correct. So, the data is collected for all-cause mortality, then we back-trace to what could be the cause of this mortality and came up with a list of 15 things, excluding the reverse causality stuff.  Hence GBD (and us) are using the RRs inconsistent with the data.

Reply:
revised as: 

In the risk appendix of GBD 2017, it says the relative risks for lbwsg apply to all-cause mortality rates. However, we are choosing to apply the RRs *only* to the causes that GBD believe lbswg affects (the list of affected causes). Ours approach and GBD's may be inconsistent with what the RRs from the data are for.

Reply:
Hmm... I don't think that GBD is actually making the assumption that "all of the excess mortality that was measured in the RR's are theoretically all coming from these plausible blue causes"... rather they are making the conservative decision not to include these causes in their LBWSG PAF because they did not have enough evidence to conclude that they were causally related (because of concerns with reverse causality)... I think we just phrase this too strongly as is and instead could say something like "GBD evaluated the causes for which LBWSG causes increased mortality and concluded that this was the cases for the blue causes. They assumed that the relative risks for all-cause mortality rates by LBWSG category applied equally to mortality rates from each of these blue causes only and did not apply to any other GBD causes in order to calculate the population attributable burden due to LBWSG; in other words, the conservatively ignored the potential impact of LBWSG on mortality due to causes that did not meet their causal criteria."